Doppler.HtmlEditorApi/Repositories.DopplerDb/DopplerCampaignContentRepository.cs

```diff
@@ -147,14 +130,6 @@ public async Task UpdateCampaignContent(string accountName, ContentData content)
         };
 
         await _dbContext.ExecuteAsync(updateContentQuery);
-
-        if (content is BaseHtmlContentData baseHtmlContentData)
-        {
```

Comment:
This condition has been removed, is it fine?

Reply:
I think its fine because contentRow is defined as BaseHtmlContentData
https://github.com/FromDoppler/doppler-html-editor-api/blob/fe6914ddf1208e673d1006cec2c41981b4ce026a/Doppler.HtmlEditorApi/Controllers/CampaignsController.cs#L113

Reply:
You are right, it is not necessary here.

But reading this code, I find another improvement point: `previewImage` is no longer part of the `contentRow`. What do you think about removing this property from `BaseHtmlContentData`? (it could be another refactor step)

Next, it is not a refactor, but I think that it could be a minor change that could help us to keep compatibility in future steps, what do you think about updating the preview image in the database only when `campaignContent.previewImage != null`?

Let me know if this comment is not clear.

Reply:
Great suggestion!